src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -420,12 +442,21 @@ class SignUpModalController {
       return
     }
 
+    if (context.formName === 'enroll-authenticator') {
+      // Verification step page
+      this.$scope.$apply(() => {
+        this.currentStep = 4
+      })
```

Comment:
```
this.currentStep = 4
```
This should be added to the `postSubmit` function as that is the function that runs when the data is successfully saved

Reply:
* Go through the sign up flow to create an account, but don't enter the verification code.
* Close the sign up modal.
* Open the sign up modal again.

If you follow those steps, Okta will immediately take you to the verification step. I was thinking that `afterRender` would be the best place to detect that we're on the `enroll-authenticator` form. It can't just go in `postSubmit` because `postSubmit` isn't called when the user opens the sign up modal the second time. But please let me know if there is a better place to put this.